src/common/xBuild.ts

```diff
@@ -6,7 +6,7 @@ type XmlElement = {
   [tag: string]: any;
 };
 
-const nsMap = mapKeys(namespaces, (v, k) => {
+const nsMap = () => mapKeys(namespaces, (v, k) => {
```

Comment:
Is the change to make this a function that returns the `nsMap` object because it was being modified in the responses over multiple `multistatus` function calls?

Reply:
Yes. The changes modified the map inplace and by creating a new map each time, this is prevented.